bundlepath.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package charmrepo
+
+import (
+	"os"
+	"path/filepath"
+
+	"gopkg.in/errgo.v1"
+	"gopkg.in/juju/charm.v6-unstable"
+)
+
+// NewBundleAtPath creates and returns a bundle at a given path.
```

Comment:
// NewBundleAtPath creates and returns a bundle at a given path
// and a URL that describes it.

?